chesster.js

```diff
@@ -644,6 +647,78 @@ controller.hears([
     });
 });
 
+controller.hears([
+    'pairing ([a-zA-Z0-9]+) ([1-8])'
```

Comment:
You don't need to do all of the refactoring work that I suggest here. But I would like to see this also support the @lakinwecker syntax if possible.  People will use that often. And support dashes at the very least for people like @r-mena